conf/versions.env

```diff
@@ -1,4 +1,4 @@
-ARLAS_WUI_VERSION=gisaia/arlas-wui:26.1.0-rc.2-no-analytics
+ARLAS_WUI_VERSION=gisaia/arlas-wui:26.1.0-rc.3-no-analytics
```

Comment:
Official 26.1.0 release is out for wui, hub and builder

Reply:
ok, I upgrade